### ISSUE TITLE
Dread Pirate Ring not granting the 5% additional experience

### ISSUE THREAD

**Issue by boujii** [dalaranwow/dalaran-wow#5514]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)
Dread Pirate Ring does not give any additional experience points for either NPC kills or quest achievements,
**Expected behavior**: Tell us what should happen instead
Experience from killing mobs and completing questing should grant an additional 5% experience.

**Steps to reproduce the problem**:

1. Ring off - Kill NPC - Ring on - Kill NPC
2. Ring off - Complete Quest - Ring on - Complete Quest
3. 

**Include proofs for this behavior**
Comments in World chat says it does not work for them either.
![screenshot_2](https://user-images.githubusercontent.com/1514895/41446899-424082f8-7020-11e8-9396-7170513b082a.png)
![screenshot_3](https://user-images.githubusercontent.com/1514895/41446940-87eb4784-7020-11e8-9174-1e9f01567ffb.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
It works as intended. Please make sure that you have the item equipped.

The NPC granted 132 experience without the ring, and 138 with the item equipped (+6,6exp / +5%)

![image](https://user-images.githubusercontent.com/5057942/41447907-0426b202-7058-11e8-9196-cee200694b9a.png)

-----------------
---------------------

This quest was supposed to grant 625 experience, however it granted 656 with Dread Pirate Ring equipped (+31,25exp / +5%)
![image](https://user-images.githubusercontent.com/5057942/41447897-f8c74ee4-7057-11e8-98e7-bab7a9dd1112.png)

---------------------

Closing up.
